### ISSUE TITLE
concurent start limit

### ISSUE THREAD

**Issue by paultag** [paultag/moxie#54]
rate limit the number of jobs being brought up at once. Let's do 2 at a time at most?

**Comment by mileswwatkins** [comment]
This would be lovely; I'm concerned about the effect of every Open State in Eastern Time starting their scrapers at the same time, haha. Could tweak the `moxie-jobs` start time individually, but this seems like a much better solution.

**Comment by mileswwatkins** [comment]
Yeah, with so many jobs starting promptly at local midnight last night, we had tons of `MIA`s. @rshorey and I are going to try to knock this out.

**Comment by paultag** [comment]
Try using a blocking channel to send jobs to a central worker, and use that
for controlling concurent bringup :+1:

On Tue, Aug 18, 2015 at 4:17 PM, Miles Watkins <notifications@github.com>
wrote:

> Yeah, with so many jobs starting promptly at local midnight last night, we
> had tons of MIAs. @rshorey <https://github.com/rshorey> and I are going
> to try to knock this out.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paultag/moxie/issues/54#issuecomment-132225925>.
>



-- 
:wq
